### ISSUE TITLE
Establish annotation protocol

### ISSUE THREAD

**Issue by ktl014** [hab-spc/hab-ml#42]
### Story
As a data annotator, I want to help contribute to making the models better, so I need to be able to i) formulate a protocol for how to communicate with the ML engineers about how to make the dataset and ii) create an example test set to begin the process. This test set will be used to assess our first model's performance.

### Definition of Done
-  [ ] Workflow / Design doc / Agreed contract between data annotators and ML engineers of how the creation of training sets should be communicated
 
Due at the end of the sprint

**Comment by ktl014** [comment]
@igneel73 @zhouyuanyuan1999, we need to start discussing what's the protocol for assessing the model performance, since we have a model. Obviously part of it is assembling this test set, but i want to come to an agreement about the performance metrics we're gonna evaluate the model by and how to get annotations to Zhouyuan.

@sarahstrang is gonna be in charge of organizing this protocol. Expect to hear from her about this and contributing to a google doc of some sort.
